PowerEditor/src/MISC/Common/Common.cpp

```diff
@@ -747,4 +747,37 @@ generic_string stringJoin(const std::vector<generic_string> &strings, generic_st
 		}
 	}
 	return joined;
+}
+
+bool str2Clipboard(const TCHAR *str2cpy, HWND hwnd)
+{
+	if (!str2cpy)
+		return false;
+
+	int len2Allocate = lstrlen(str2cpy) + 1;
+	len2Allocate *= sizeof(TCHAR);
+	unsigned int cilpboardFormat = CF_TEXT;
+
+	cilpboardFormat = CF_UNICODETEXT;

```

Comment:
I noticed that you dropped the `#ifdef UNICODE`/`#endif`....did you mean:

```
unsigned int clipboardFormat = CF_TEXT;
clipboardFormat |= CF_UNICODETEXT;
```

or:

```
clipboardFormat = CF_UNICODETEXT;
```
...or something else?